src/metkit/mars/StepRange.cc

```diff
@@ -102,7 +102,8 @@ void StepRange::print(std::ostream& s) const
         eckit::Time t{static_cast<long>(to_*3600.), true};
 
         TimeUnit unit = std::min(maxUnit(f), maxUnit(t));
-        s << canonical(f, unit) << '-' << canonical(t, unit);
+        // s << canonical(f, unit) << '-' << canonical(t, unit);
+        s << canonical(f) << '-' << canonical(t);
```

Comment:
Why are we no longer using unit here?

Reply:
the initial request was to use the same unit for stepranges,
so 30m-1h ==> 30m-60m
 
with the to-by, we have horrible sequences:
0-6/to/3-9/by/30m ==> 0-6/30m-390m/1-7/90m-450m/2-8/150m-510m/3-9
with the proposed change we have:
0-6/to/3-9/by/30m ==> 0-6/30m-6h30m/1-7/1h30m-7h30m/2-8/2h30m-8h30m/3-9